Phaethon/Tests/IntegrationTests/LoginTest.cs

```diff
@@ -1,78 +1,78 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Collections.Immutable;
-using System.Data.Entity;
-using System.Data.Entity.Core.Objects;
 using System.Linq;
 using System.Net;
 using System.Net.Http;
-using System.Runtime.ExceptionServices;
 using System.Text;
 using System.Threading.Tasks;
 using Core.Model;
 using InternalApi.DataAccess;
-using Newtonsoft.Json;
 using NUnit.Framework;
 using System.Security.Cryptography;
-using InternalApi.DataManagement;
 
 namespace Tests.IntegrationTests
 {
-
    public class LoginTest : InternalTestFakeServerBase
     {
-        private Login userModel;
-        private Login userModel2;
-        private int count1;
-        private int count2;
-        private string name;
-        private string name2;
-        private string OGPass;
-        private LoginDa loginDa;
+        private Login _userModel;
+        private Login _userModel2;
+        private string _name;
+        private string _name2;
+        private string _originalPass;
+        private LoginDa _loginDa;
 
         private void InitializeData()
```

Comment:
Add `[SetUp]` attribute to this method (just like you did with TearDown, this way you wont need to call this method in everry test 😸